sourced/ml/__main__.py

```diff
@@ -37,6 +38,25 @@ def one_arg_parser(*args, **kwargs) -> argparse.ArgumentParser:
     return arg_parser
 
 
+def add_spark_args(my_parser):
+    my_parser.add_argument(
+        "-s", "--spark", default="local[*]", help="Spark's master address.")
+    my_parser.add_argument(
+        "--config", nargs="+", default=[], help="Spark configuration (key=value).")
+    my_parser.add_argument(
+        "--package", nargs="+", default=[], help="Additional Spark package.")
+    my_parser.add_argument(
+        "--spark-local-dir", default="/tmp/spark", help="Spark local directory.")
```

Comment:
Please include a handy memory config:

```
--config spark.executor.memory=4G --config spark.driver.memory=10G --config spark.driver.maxResultSize=4G
```

should be replaced with

```
-m / --memory 4,10,4
```

Numbers are floats separated by commas.